src/Symfony/Component/CssSelector/Parser/Tokenizer/TokenizerPatterns.php

```diff
@@ -49,22 +49,22 @@ public function __construct()
         $this->identifierPattern = '-?(?:'.$this->nmStartPattern.')(?:'.$this->nmCharPattern.')*';
         $this->hashPattern = '#((?:'.$this->nmCharPattern.')+)';
         $this->numberPattern = '[+-]?(?:[0-9]*\.[0-9]+|[0-9]+)';
-        $this->quotedStringPattern = '([^\n\r\f%s]|'.$this->stringEscapePattern.')*';
+        $this->quotedStringPattern = '([^\n\r\f\\\\%s]|'.$this->stringEscapePattern.')*';
     }
 
     public function getNewLineEscapePattern(): string
     {
-        return '~^'.$this->newLineEscapePattern.'~';
+        return '~'.$this->newLineEscapePattern.'~';
```

Comment:
https://github.com/scrapy/cssselect/blob/ddd9784977fca6e7da4439d837e4f510f1f10638/cssselect/parser.py#L930

Check the 3 escape`"_sub_*` "functions" and how they are used. All occurrences are replaced.